raw/graphics/graphics_aikoncwd_professions.txt

```diff
@@ -37,3 +47,17 @@ graphics_aikoncwd_professions
 	[PROPHET:AIKONCWD_GOBLIN:2:0:AS_IS:DEFAULT]
 	[PILGRIM:AIKONCWD_GOBLIN:0:1:AS_IS:DEFAULT]
 	[MONK:AIKONCWD_GOBLIN:1:1:AS_IS:DEFAULT]	
+
+[CREATURE_GRAPHICS:HUMAN]
+	[CRIMINAL:AIKONCWD_HUMAN:0:0:AS_IS:DEFAULT]
+	[PEDDLER:AIKONCWD_HUMAN:1:0:AS_IS:DEFAULT]
+	[PROPHET:AIKONCWD_HUMAN:2:0:AS_IS:DEFAULT]
+	[PILGRIM:AIKONCWD_HUMAN:0:1:AS_IS:DEFAULT]
+	[MONK:AIKONCWD_HUMAN:1:1:AS_IS:DEFAULT]	
+
+[CREATURE_GRAPHICS:FOREST]

```

Comment:
Check if this is correct! :bug:

Reply:
It should be [CREATURE_GRAPHICS:ELF] instead of [CREATURE_GRAPHICS:FOREST]